### PR TITLE
Add GDL to serial packages

### PR DESCRIPTION
**Pull request by rmsds**
Signed-off-by: Ricardo Silva <ricardo.silva@epfl.ch>

### CODE REVIEW

**Comment by rmsds** [comment]
Blocked by https://github.com/epfl-scitas/spack/pull/192.

**Comment by rmsds** [comment]
Please do not merge yet, I still need to confirm:

- [x] whether we need the python interface, yes: https://github.com/epfl-scitas/spack/pull/195
- [x] populate the blacklist with all the new dependencies.

File: configuration/modules.yaml [hunk 1]
```diff
@@ -103,30 +103,49 @@ modules:
 
     blacklist: [
       '%gcc@4.8.5',
+      'atk',
       'autoconf',
       'automake',
       'boost~mpi',
       'cairo',
+      'cairo',
```
Comment (nrichart) [inline configuration/modules.yaml]: duplicate

File: configuration/modules.yaml [hunk 2]
```diff
@@ -103,30 +103,49 @@ modules:
...
       'flex',
       'freetype',
+      'gdk-pixbuf',
       'gettext',
+      'ghostscript',
+      'ghostscript-fonts',
       'glib',
+      'gobject-introspection',
+      'graphicsmagick',
+      'graphviz',
```
Comment (nrichart) [inline configuration/modules.yaml:121]: no need to blacklist (contains usefull tools)

File: configuration/modules.yaml [hunk 3]
```diff
@@ -174,14 +193,18 @@ modules:
       'py-virtualenv',
       'python~tk',
       'readline',
+      'shared-mime-info',
       'suite-sparse ^metis~real64',
+      'swig',
```
Comment (nrichart) [inline configuration/modules.yaml]: also usefull no need to blacklist

File: paien.yaml [hunk 4]
```diff
@@ -430,6 +431,14 @@ packages:
...
       # FIXME: Concretization errors on gl vdep
       # - opencv@3.4.1 +qt+zlib+tiff+vtk+png+jpeg+imgproc+highgui build_type=Release ^qt@4.8.6 +gtk+opengl+webkit ^mesa
+      - gdl~wx~x11~hdf4+hdf5+embed-python+python+proj+graphicsmagick+openmp
+        ^eigen@3.3.4~fftw~suitesparse~metis~scotch~mpfr
+        ^fftw@3.3.7~mpi~openmp+fma simd=sse2,avx,avx2,avx512
+        ^netcdf@4.6.1~mpi
+        ^hdf5@1.10.1+szip~mpi+hl+fortran+cxx
+        ^python@2.7.14+tk
+        ^py-numpy+blas+lapack@1.14.3
+        ^openblas@0.2.20 threads=openmp
```
Comment (nrichart) [inline paien.yaml]: create a new section gnu_python_2_only_lapack_openmp

**Review by nrichart — CHANGES_REQUESTED** [review]

File: configuration/modules.yaml (hunk 2 above)
Comment (rmsds) [reply configuration/modules.yaml:121]: were you thinking of graphicsmagick? graphviz is installed in the system (it's for the dot graph language), graphicsmagick is the one with the convert subcommands,etc

**Review by alalazo — APPROVED** [review]
LGTM as soon as Jenkins gives green light.

**Pull request merged**